docs/README.md

```diff
@@ -61,12 +61,13 @@ For example:
 },
 ```
 
-Every block in this app only has two props in common:
+Every block in this app only has three props in common:
 
 | Prop name          | Type      |  Description | Default value |
 | --------------------| ----------|--------------|---------------|
 | `markers`           |`[string]` | IDs of your choosing to identify the block's exported messages and customize them using the admin's Site Editor. Learn how to use them accessing the documentation on [Using the Markers prop to customize a block's message](https://vtex.io/docs/recipes/style/using-the-markers-prop-to-customize-a-blocks-message). |`[]`|
 |  `blockClass`  |  `string`  |  Block  ID  of your choosing to  be  used  in [CSS  customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property).  |  `undefined`  |
+|  `message`  |  `string`  |  Message variables exported by each block that can be edited in Site Editor as well  |  `undefined`  |
```

Comment:
```suggestion
|  `message`  |  `string`  |  Defines the block's rendered text i.e. the block message. Instead of using this prop, you can also define which text message a block will render on the UI using the admin's Site Editor and the `markers` prop.  |  `undefined`  |
```